src/main/java/org/hmhb/config/DefaultConfigService.java

```diff
@@ -28,24 +28,8 @@ public DefaultConfigService(
     ) {
         requireNonNull(environment, "environment cannot be null");
 
-        this.publicConfig = new PublicConfig(
-                environment.getProperty("google.oauth.client.id"),
-                environment.getProperty("hmhb.url.prefix"),
-                environment.getProperty("hmhb.program.startYear.minValue", Integer.class),
-                environment.getProperty("hmhb.program.streetAddress.maxLength", Integer.class),
-                environment.getProperty("hmhb.program.city.maxLength", Integer.class),
-                environment.getProperty("hmhb.program.goal.maxLength", Integer.class),
-                environment.getProperty("hmhb.program.outcome.maxLength", Integer.class),
-                environment.getProperty("hmhb.program.name.maxLength", Integer.class),
-                environment.getProperty("hmhb.program.otherProgramArea.explanation.maxLength", Integer.class),
-                environment.getProperty("hmhb.organization.name.maxLength", Integer.class)
-        );
-
-        this.privateConfig = new PrivateConfig(
-                environment.getProperty("google.oauth.client.secret"),
-                environment.getProperty("hmhb.jwt.domain"),
-                environment.getProperty("hmhb.jwt.secret")
-        );
+        this.publicConfig = new PublicConfig(environment);
```

Comment:
This cleans up all of the tests having intimate details of all config (and removes the obnoxious constructor params.  However, it does come at the cost of misconfig errors being delayed (I don't expect anyone to really modify the application.properties file anyway though).